admin/src/js/services/create-user.js

```diff
@@ -20,6 +20,7 @@
        * @param      {Object}  updates        Updates you wish to make
        */
       const createSingleUser = (updates) => {
+        URL = '/api/v3/users';
```

Comment:
Because this is a singleton, we will end up changing this value for all the following calls. 
I think we should use a scoped variable here, instead of overwriting the global one. Or have two global variables.